iceoryx_dds/CMakeLists.txt

```diff
@@ -133,7 +142,10 @@ target_link_libraries(iox-gw-dds2iceoryx
     iceoryx_posh::iceoryx_posh
     ${PROJECT_NAMESPACE}::iceoryx_dds
 )
-
+set_target_properties(iox-gw-dds2iceoryx PROPERTIES
```

Comment:
You can safe code by adding multiple targets to it and delete the two statements above
```
set_target_properties(iceoryx_dds iox-gw-iceoryx2dds iox-gw-dds2iceoryx PROPERTIES
    CXX_STANDARD_REQUIRED ON
    CXX_STANDARD ${ICEORYX_CXX_STANDARD}
)
```

Reply:
done